### PR TITLE
Update of RuleType Configuration Cheat Sheet

### PR DESCRIPTION
**Pull request by nsano-rururu**
## Description

Update RuleType Configuration Cheat Sheet

**Fix**

- (boolean, no default) → (boolean, default False)
- `alert_on_missing_fields` → `alert_on_missing_field`

**Add**

- Metric Aggregation
- Spike Aggregation
- Percentage Match

## Checklist

- [x] I have reviewed the [contributing guidelines](https://github.com/jertel/elastalert2/blob/master/CONTRIBUTING.md).
- [ ] I have included unit tests for my changes or additions.
- [x] I have successfully run `make test-docker` with my changes.
- [ ] I have manually tested all relevant modes of the change in this PR.
- [x] I have updated the [documentation](https://elastalert2.readthedocs.io).
- [x] I have updated the [changelog](https://github.com/jertel/elastalert2/blob/master/CHANGELOG.md).

### CODE REVIEW

File: docs/source/ruletypes.rst [hunk 1]
```diff
@@ -153,67 +153,111 @@ Rule Configuration Cheat Sheet
...
+|``alert_on_new_data`` (boolean, default False)         |        |           |           |        |           |   Opt |          |        |           |                  |                 |                |
++-------------------------------------------------------+--------+-----------+-----------+--------+-----------+-------+----------+--------+-----------+------------------+-----------------+----------------+
+|``threshold_ref`` (int, no default)                    |        |           |           |        |           |   Opt |          |        |           |                  |                 |                |
++-------------------------------------------------------+--------+-----------+-----------+--------+-----------+-------+----------+--------+-----------+------------------+-----------------+----------------+
+|``threshold_ref`` (number, no default)                 |        |           |           |        |           |       |          |        |           |                  |  Opt            |                |
```
Comment (jertel) [inline docs/source/ruletypes.rst:199]: Did you intend to duplicate the threshold_ref and threshold_cur values? One is labeled as a number, the other as an int.

Comment (nsano-rururu) [reply docs/source/ruletypes.rst:199]: look  schema.yaml

Spike
https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml#L131
```yaml
      threshold_ref: {type: integer}
      threshold_cur: {type: integer}
```

Spike Aggregation
https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml#L147
```yaml
      threshold_ref: {type: number}
      threshold_cur: {type: number}
```

Comment (nsano-rururu) [reply docs/source/ruletypes.rst:199]: The difference between number and integer is that number is allowed after the decimal point. integer is an integer only

Comment (nsano-rururu) [reply docs/source/ruletypes.rst:199]: It seems that such schema.yaml is set for the above reason.

**Review by jertel — APPROVED** [review]

**Pull request merged**